website/static/css/site.css

```diff
@@ -1946,6 +1946,10 @@ span#profileFullname{
     padding-bottom: 20px;
 }
 
+/*Dashboard */
+.col-sm-5 .tab-content .addon-widget-body{
```

Comment:
This is a brittle selector; what happens if we change, e.g. from `.col-sm-5` to `.col-sm-4`?